components/Chips/src/MDCChipView.h

```diff
@@ -0,0 +1,169 @@
+ UIContentSizeCategory is changed.
+
+ This property is modeled after the adjustsFontForContentSizeCategory property in the
+ UIConnectSizeCategoryAdjusting protocol added by Apple in iOS 10.0.
+
+ If set to YES, this button will base its text font on MDCFontTextStyleButton.
+
+ Default value is NO.
+ */
+@property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)
+    BOOL mdc_adjustsFontForContentSizeCategory UI_APPEARANCE_SELECTOR;
+
+/*
+ A color used as the chip's @c backgroundColor for @c state.
+
+ @param state The state.
+ @return The background color.
+ */
+- (nullable UIColor *)backgroundColorForState:(UIControlState)state;
+
+/*
+ A color used as the chip's @c backgroundColor.
+
+ If left unset or reset to nil for a particular control state, then a default blue color is used.
+
+ @param backgroundColor The background color.
+ @param state The state.
+ */
+- (void)setBackgroundColor:(nullable UIColor *)backgroundColor forState:(UIControlState)state
+    UI_APPEARANCE_SELECTOR;
+
+/*
+ Returns the border color for a particular control state.
+
+ @param state The control state to retrieve the elevation.
+ @return The border color for the requested state.
+ */
+- (nullable UIColor *)borderColorForState:(UIControlState)state;
+
+/*
+ Sets the border color for a particular control state.
+
+ @param borderColor The border color.
+ @param state The state.
+ */
+- (void)setBorderColor:(nullable UIColor *)borderColor forState:(UIControlState)state
+    UI_APPEARANCE_SELECTOR;
+
+/*
+ Returns the border width for a particular control state.
+
+ @param state The control state to retrieve the elevation.
+ @return The border width for the requested state.
+ */
+- (CGFloat)borderWidthForState:(UIControlState)state;
+
+/*
+ Sets the border width for a particular control state.
+
+ @param borderWidth The border width.
+ @param state The state.
+ */
+- (void)setBorderWidth:(CGFloat)borderWidth forState:(UIControlState)state UI_APPEARANCE_SELECTOR;
+
+/*
+ Returns the elevation for a particular control state.
+
+ @param state The control state to retrieve the elevation.
+ @return The elevation for the requested state.
+ */
+- (CGFloat)elevationForState:(UIControlState)state;
+
+/*
+ Sets the elevation for a particular control state.
+
+ @param elevation The elevation to set.
+ @param state The state to set.
+ */
+- (void)setElevation:(CGFloat)elevation forState:(UIControlState)state UI_APPEARANCE_SELECTOR;
+
+/*
+ Returns the title color for a particular control state.
+
+ @param state The control state to retrieve the elevation.
+ @return The title color for the requested state.
+ */
+- (nullable UIColor *)titleColorForState:(UIControlState)state;
+
+/*
+ Sets the title color for a particular control state.
+
+ @param titleColor The title color.
+ @param state The state.
+ */
+- (void)setTitleColor:(nullable UIColor *)titleColor forState:(UIControlState)state
```

Comment:
Same suggestion to follow [UIButton's convention](https://developer.apple.com/documentation/uikit/uibutton/1623993-settitlecolor?language=objc) when handling values for state.